src/KM_Maps.pas

```diff
@@ -1068,5 +1080,20 @@ procedure TTMapsCacheUpdater.ProcessMap(aPath: UnicodeString; aFolder: TMapFolde
 end;
 
 
+{Utility methods}
+//Return TMapFolder for specified aFolderName
+function GetMapFolder(aFolderName: UnicodeString): TMapFolder;
+var F: TMapFolder;
+begin
+  for F := Low(TMapFolder) to High(TMapFolder) do
+    if aFolderName = MAP_FOLDER[F] then
+    begin
+      Result := F;
+      Exit;
+    end;
+  raise Exception.Create('Unexpected map folder: ' + aFolderName);
```

Comment:
Using Save from the F11 main menu might trigger this exception? It's not really unexpected to save outside of Maps and MapsMP.